OpenOversight/app/__init__.py

```diff
@@ -43,12 +43,12 @@ def create_app(config_name="default"):
     from .models import db
 
     bootstrap.init_app(app)
-    mail.init_app(app)
+    csrf.init_app(app)
     db.init_app(app)
-    login_manager.init_app(app)
+    GmailClient()
```

Comment:
This is added to the `__init__` file so that the build fails if the configuration file isn't there.